addons/mass_mailing/static/src/views/mailing_contact_view_kanban.js

```diff
@@ -1,28 +0,0 @@
-/** @odoo-module **/
-
-import { KanbanController } from "@web/views/kanban/kanban_controller";
-import { kanbanView } from '@web/views/kanban/kanban_view';
-import { registry } from '@web/core/registry';
-import { useService } from "@web/core/utils/hooks";
-
-export class MailingContactController extends KanbanController {
-    async setup() {
-        super.setup();
-        this.actionService = useService("action");
-    }
-
-    onImport() {
-        const context = this.props.context;
-        const actionParams = { additionalContext: context };
-        if (!context.default_mailing_list_ids && context.active_model === 'mailing.list' && context.active_ids) {
```

Comment:
Hello @FrancoisGe @aab-odoo 

Maybe it was discussed with Luc, but this code actually has a purpose.
If you go into a mailing list, then click on the "Recipients" stat button, then go into the kanban view and click "Import", it will auto-populate the "active" Mailing List.

(It used to work in the list view as well but I think it broke with your previous task?).

Maybe you guys agreed with Luc/Damien that this feature is not necessary, but I wanted to make sure.

Cheers!